test/e2e/framework/helper/certificates.go

```diff
@@ -82,175 +75,6 @@ func (h *Helper) WaitForCertificateNotReadyUpdate(cert *cmapi.Certificate, timeo
 	return h.handleResult(cert.Namespace, cert.Name, result, err)
 }
 
-// ValidateIssuedCertificate will ensure that the given Certificate has a
```

Comment:
Is this function gone because it is unused? (I did search for usages and found none).

It doesn't look correct at first glance, so axe it.